src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -37,7 +37,6 @@ $pf-v6--include-toolbar-breakpoints: true !default;
   --#{$toolbar}__content--RowGap: var(--pf-t--global--spacer--sm);
   --#{$toolbar}__content--PaddingBlock: var(--pf-t--global--spacer--md);
   --#{$toolbar}__content--PaddingInline: var(--#{$toolbar}--PaddingInline);
-  --#{$toolbar}__content--PaddingBlockStart: 0;
```

Comment:
This works well for `page` padding. It will need to be isolated the `page` or updated otherwise.

![Screenshot 2024-06-21 at 1 24 28 PM](https://github.com/patternfly/patternfly/assets/5385435/4daf5f1b-d8dc-486c-98d6-e1fae50b55b7)